vendor/github.com/aws/aws-sdk-go/aws/BUILD

```diff
@@ -1,54 +0,0 @@
-load("@io_bazel_rules_go//go:def.bzl", "go_library")
```

Comment:
Was the deletion of these BUILD files on purpose? (maybe run hack/update-bazel.sh?)